client/init.js

```diff
@@ -18,25 +22,46 @@ addEventListener('load', function() {
 
 		form.querySelector('button').disabled = false
 	}
-
+	/**
+	 *
+	 * @param  {string} name The name of the new user, the input. Should probably
+	 *                       validate this.
+	 * @return {stuff}      stuff, not quite sure what stuff is yet.
```

Comment:
Here, stuff is an array of two resolved promises. One is the comms object, the other is a map. `stuff` is definitely a bad name for this variable, I'm going to change it, but I can't change that this needs to return two things.